src/compiler/scala/tools/nsc/Global.scala

```diff
@@ -1503,7 +1508,13 @@ class Global(var currentSettings: Settings, var reporter: Reporter)
       }
       symSource.keys foreach (x => resetPackageClass(x.owner))
 
-      informTime("total", startTime)
+      if (timePhases) {
+        statistics.stopTimer(totalCompileTime, startTotal)
+        informTime("total", totalCompileTime.nanos)
+        inform("*** Cumulative timers for phases")
+        for (q <- statistics.allQuantities if q.phases.nonEmpty && q.showAt(GlobalPhaseName))
```

Comment:
Nit: the guard `q.phases.nonEmpty && q.showAt(GlobalPhaseName)` is non-obvious to understand - how about the more explicit `q.phases == Seq(GlobalPhaseName)`?

Reply:
We can use this because `GlobalPhaseName` will always be the only phase, but it's more restrictive semantically.